lbz/response.py

```diff
@@ -0,0 +1,45 @@
+#!/usr/local/bin/python3.8
```

Comment:
why did You split communication into 2 files ?

Reply:
To avoid circular imports